service_objects/services.py

```diff
@@ -19,3 +19,20 @@ def service_clean(self):
 
     def process(self):
         raise NotImplementedError()
+
+
+class ModelService(forms.ModelForm):
```

Comment:
Basing ModelService off ModelForm comes with some baggage of ModelForms functionality tied tightly to it controlling a model's create/update/delete cycle.  Plus, this implementation duplicates everything between Service and ModelService.

A better approach would be:

```python
from django.forms import models

class ModelService(Service, metaclass=models.ModelFormMetaclass):
	pass

```
`ModelFormMetaclass` metaclass is what gives the functionality of looking at a model and auto-magically creating fields from it.  That's really the only functionality you need for ModelService.  And this way, changes to Service is reflected in ModelService without duplication.